scene/3d/physics/soft_body_3d.cpp

```diff
@@ -71,15 +75,70 @@ void SoftBodyRenderingServerHandler::clear() {
 }
 
 void SoftBodyRenderingServerHandler::open() {
-	write_buffer = buffer.ptrw();
+	write_buffer = buffer_curr.ptrw();
 }
 
 void SoftBodyRenderingServerHandler::close() {
 	write_buffer = nullptr;
 }
 
-void SoftBodyRenderingServerHandler::commit_changes() {
-	RS::get_singleton()->mesh_surface_update_vertex_region(mesh, surface, 0, buffer);
+void SoftBodyRenderingServerHandler::fti_pump() {
+	if (buffer_prev.is_empty()) {
+		buffer_prev.resize(buffer_curr.size());
+	}
+	SWAP(buffer_prev, buffer_curr);
+	aabb_prev = aabb_curr;
+}
+
+void SoftBodyRenderingServerHandler::commit_changes(real_t p_interpolation_fraction) {
+	real_t f = p_interpolation_fraction;
+	AABB aabb_interp = aabb_curr;
+
+	if (buffer_interp.is_empty()) {
+		buffer_interp.resize(buffer_curr.size());
+	}
```

Comment:
No need to do this if physics interpolation is off.
We want things to be zero cost (or close to) for projects not using physics interpolation, both in terms of processing and memory use.